hibike/devices/YogiBear/motor.cpp

```diff
@@ -22,12 +24,12 @@ void motorSetup() {
 
 void motorEnable() {
   clearFault();
-  digitalWrite(enable_pin, HIGH);
+  analogWrite(enable_pin, analogHIGH); /* digital to analog where HIGH corresponds to 255*/
```

Comment:
So why did we change these lines from digitalWrite to analogWrite?

There's no reason to.  If you analogWrite to a pin, you'll have to still configure it as an input before you analog read it.  You can't configure the pin to be an analog/digital pin...

Reply:
I should also point this out.

The enable_pin is pulled up with a resistor on the red board.  So I think we should actually put this pin into a High Impedance state (aka make it an analog/digital input) to enable the pin, not write the value high.

Let's pretend that we could both do analog_writes and analog_reads at the same time (we can't).  If we could, once we enabled the motor, what analog value would we see?  It'd always be high: regardless of what the motor's trying to do, even if it's trying to pull the line low to signal it's in an error state.